examples/save_disp.py

```diff
@@ -0,0 +1,68 @@
+import os
+from pathlib import Path
+from packaging import version
+
+import torch
+import numpy as np
+from PIL import Image
+
+
+def load_image(imfile):
+    """Utility function to load image files as tensors"""
```

Comment:
Please adhere to our style guide.
```suggestion
def load_image(imfile: ?) -> ?:
    """ Utility function to load image files as tensors.

	Args:
		...

	Raise:
		...

	Returns:
		...
    """
```